tests/packagedcode/test_maven.py

```diff
@@ -465,7 +465,7 @@ def test_get_license_detections_with_unknown_url_known_comments(self):
             {'name': 'mit'}
         ]
         _detections, result = get_license_detections_and_expression(declared_license)
-        expected = 'apache-2.0 AND mit'
+        expected = 'apache-2.0 AND unknown AND mit'
```

Comment:
This is a regression and is a problem!

Reply:
There are some issues wrt. Maven and package license detection that are tracked separately now:
- https://github.com/aboutcode-org/scancode-toolkit/issues/4262
- https://github.com/aboutcode-org/scancode-toolkit/pull/4261
- https://github.com/aboutcode-org/scancode-toolkit/issues/4245